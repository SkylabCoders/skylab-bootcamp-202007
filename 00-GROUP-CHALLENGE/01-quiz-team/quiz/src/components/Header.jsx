import React from 'react';
import { NavLink } from 'react-router-dom';
import '../css/header.scss';
import * as ROUTES from './../config/routes';


function Header() {
	return (
		<>
			<div className="header border border-info">
				<div className="inner__header">
						<div className="logo__header">
							<img className="img__header" src="https://storage.needpix.com/rsynced_images/quiz-2074324_1280.png" alt="quiz logo" />
						</div>
					<nav className="nav">
						<ul className="nav__list">
							<li>
								<NavLink className="nav__item" activeClassName="my-active-class link" to={ROUTES.HOME}>
									Home
								</NavLink>
							</li>
							<li> {' | '} </li>
							<li>
								<NavLink className="nav__item" activeClassName="my-active-class link" to={ROUTES.LEADERBOARD}>
									Leaderboard
								</NavLink>
							</li>
						</ul>
					</nav>
					<NavLink activeClassName="my-active-class" to="/login">
						<div className="login__icon"><img src={require('../assets/user_icon.png')} /> </div>
					</NavLink>
					<NavLink activeClassName="my-active-class" className="login__button__link" to={ROUTES.LOGIN}>
						<button type="button" className="login__button">Login</button>
					</NavLink>
				</div>
			</div>
		</>
	)
}

export default Header;

