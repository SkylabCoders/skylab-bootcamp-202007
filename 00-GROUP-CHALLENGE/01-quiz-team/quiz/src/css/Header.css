.header {
	width: 100%;
	background-color: #ccc7c7;
	display: flex;
	flex-direction: row;
	justify-content: space-around;
	align-items: center;
	box-sizing: border-box;
	padding: 10px;
}

.inner__header{
    width: 90%;
    display: flex;
    flex-direction: row;
    justify-content: space-around;
    align-items: center;
}
nav{
    margin: 0 20px;
    width: 100%;
}

.logo__header{
    display: none;
}

.img__header{
    width: 25%;
}

button{
    margin: 0 5px;
}

@media (min-width: 680px) and (max-width: 1024px) {
	.logo__header{ 
        width: 90%;  
        justify-content: center;
        display: contents;
    }
    
    .inner__header{
        width: 80%;
        justify-content: center;
        display: contents;
    }

    nav{
        display: flex;
        flex-direction: row;
        justify-content: center;
        align-items: center;
    }

    .img__header{
        width: 25%;
    }

}

@media (min-width: 1024px) {
    .logo__header{ 
        width: 80%;  
        justify-content: center;
        display: contents;
    }
    
    .inner__header{
        width: 80%;
        justify-content: center;
        display: contents;
    }

    .img__header{
        width: 20%;
    }

    nav{
        display: flex;
        flex-direction: row;
        justify-content: left;
        align-items: center;
    }
}

