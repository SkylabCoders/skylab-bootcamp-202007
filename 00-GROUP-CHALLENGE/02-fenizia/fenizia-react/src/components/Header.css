* {
    margin: 0;
    padding: 0;
    box-sizing: border-box;
}

.main-nav {
    background-color: #fff;
    display: flex;
    align-items: center;
    width: 100%;
    height: 80px;
    border-bottom: 1px solid rgb(209, 209, 209);
    position: fixed;
    top: 0;
    z-index: 80;
}

.icon__hamburger,
.icon__search,
.icon__login,
.icon__bag {
    display: inline-block;
    align-items: center;
    width: 50px;
    height: 50px;
    margin-bottom: 2px;
}

.hamburger {
    position: relative;
}

    .hamburger:hover .hamburger__products {
        display: flex;
    }

.hamburger__products {
    position: absolute;
    display: flex;
    flex-direction: column;
    background-color: #fff  ;
    width: 150px;
    height: auto;
    padding: .5rem;
    top: 50px;
    left: .8rem;
    display: none;
}

.novels,
.childish {
    color: #000;
    font-size: 1.1rem;
    text-decoration: none;
    padding: .3rem;
}

    
    .hamburger__products a:hover {
        color: #ffcc00;
    }



.icon__hamburger {
    background-image: url(../icons/hamburger.svg);
    background-position: center;
    background-size: 1.5em;
    background-repeat: no-repeat;
}  


.icon__logo {
    display: inline-block;
    flex-grow: 2;
    background-image: url(../img/logo__fenizia.svg); 
    background-position: center;   
    background-repeat: no-repeat;
    width: 150px;
    height: 60px;
    margin-left: 3rem;
} 

.icon__search {
    background-image: url(../icons/search.svg);
    background-position: center;
    background-size: 1.5em;
    background-repeat: no-repeat;
}

.icon__login{
    background-image: url(../icons/login.svg);
    background-position: center;
    background-size: 1.7em;
    background-repeat: no-repeat;
}

.icon__bag{
    background-image: url(../icons/bag.svg);
    background-position: center;
    background-size: 1.75em;
    background-repeat: no-repeat;
    margin-right: 3rem;
    text-decoration: none;
    position: relative;
}

.icon__number {
    background-color: #ffcc00;
    color: #fff;
    width: 20px;
    height: 20px;
    border-radius: 10px;
    font-size: .8rem;
    text-align: center;
    position: absolute;
    top: 5px;
    left: 5px;
    /* display: none; */
}

.icon__red {
    background-color: brown;
    padding: 3rem;
}


/* Search input */


.input__search {
    display: flex;
    width: 180px;        
    padding: .6rem;
    border: .05rem solid black;
    border-radius: .3rem;
}

/* Display None */

.main-nav-text {
    display: none;
}

/* QUERIES */

@media screen and (min-width: 990px) {
    .main-nav-text {
        display: flex;
        flex-grow: 2;
        justify-content: space-evenly;
    }

    .main-nav-text a {
        text-decoration: none;
        color: black;
        font-size: .9rem;
    }

        .main-nav-text a:hover {
            color: #ffcc00;
        }

    .icon__logo {
        flex-grow: 0;
    } 

    .input__search {
        display: flex;
        width: 180px;        
        padding: .6rem;
        border: .05rem solid black;
        border-radius: .3rem;
    }

    /* Open Menu */
    .categories {
        position: relative;
    }

    .menu__products {
        position: absolute;
        display: flex;
        flex-direction: column;
        background-color: #fff;
        width: 200px;
        height: auto;
        padding: .5rem;
        display: none;
    }

    .categories:hover .menu__products {
        display: flex;
    }

        .menu__products a:hover {
            color: #ffcc00;
        }


    /* Display None */
    .icon__hamburger {
        display: none;
    }

}


@media screen and (min-width: 1150px) {
    .input__search {
        width: 280px;          
    }
}
