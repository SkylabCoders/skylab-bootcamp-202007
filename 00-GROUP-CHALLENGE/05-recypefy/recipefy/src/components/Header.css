.nav__menu,
.nav__mobile,
.nav__medium__big {
	z-index: 90;
}

.nav__mobile,
.nav__medium__big {
	padding: 5px 5% 5px 5%;

}

.nav__mobile>a {
	color: #efee9d;
	list-style: none;
	margin: 0;
	padding: 0;
}

.burger__container {
	position: relative;
	display: flex;
	flex-direction: column;
}


.icon__burguer {
	width: 30px;
	height: auto;
	padding: 10px;
}


.burger__container:hover .desplegable__button {
	display: flex;
	flex-direction: column;
	list-style: none;
}

.desplegable__button {
	position: absolute;
	background-color: #efee9d;
	border-radius: 5px;
	display: none;
	top: 50px;
	height: 110px;
}

.desplegable__button li {
	margin: 7px;
}

.li>a,
.li>a:active,
.li>a:visited,
.li>a:-webkit-any() {
	color: rgb(75, 75, 165);
	text-decoration: none;
}

.link__header {
	text-decoration: none;
}

.login__logo {
	width: 30px;
	height: 30px;
}

.nav__brand__logo {
	width: 30px;
	height: 30px;
	border-radius: 100%;
}

.nav__mobile {
	top: 0;
	display: flex;
	flex-direction: row;
	justify-content: space-evenly;
	align-items: center;
	background-color: #50af58;
	height: 40px;
	position: fixed;
	width: 90%;
}

.nav__medium__big {
	display: none;
}


.search {
	height: 25px;
	width: 105px;
	margin: 0 10px 0 10px;
}

.main__logo {
	display: none;
}

.main__boxes {
	display: none;
}


.icon__burguer,
.login__logo {
	filter: invert(95%) sepia(17%) saturate(1084%) hue-rotate(340deg) brightness(113%) contrast(87%);
}


@media only screen and (min-width: 600px) {

	.search-button {
		padding: 5px;
		border-radius: 50px;
		background-color: #d1eaa3;
		margin: 5px;
	}

	.search-button:active,
	.search-button:link,
	.search-button:visited,
	.search-button:hover {
		color: #0f5a15;
	}


	.nav__mobile {
		display: none;
	}

	.nav__medium__big {
		top: 0;
		display: flex;
		flex-direction: row;
		justify-content: space-between;
		align-items: center;
		background-color: #50af58;
		height: 40px;
		position: fixed;
		width: 90%;
	}

	.buttons__left {
		display: flex;
		margin-left: 10px;
	}

	.link__header {
		margin-left: 10px;
		color: #efee9d;
	}

	.buttons__right {
		margin-right: 15px;

		display: flex;
		align-items: center;
	}

	.buttons__right--text {
		margin-right: 5px;
		color: #efee9d;
	}


	.section__logo {
		display: flex;
	}

	.main__logo {
		display: inline-block;
		width: 100px;
		height: 100px;
		border-radius: 100%;
	}

	.main__boxes {
		display: flex;
		margin-top: 80px;
		justify-content: space-between;
		align-items: center;
		width: 96%;
		margin: 80px 2% 0 2%;
	}

	h1 {
		display: inline-block;
		margin-left: 10px;
	}

	.section__logo {
		display: flex;
		flex-direction: row;
		background-color: #efee9d;
		justify-content: center;
		align-items: center;
		width: 40vw;
		height: 22vh;
		padding: 15px;
		border-radius: 10px;
	}

	.section__search {
		background-color: #efee9d;
		justify-content: center;
		text-align: center;
		width: 40vw;
		height: 22vh;
		padding: 15px;
		border-radius: 10px;
	}

	.section__preferences {
		display: flex;
		flex-flow: wrap row;
		justify-content: center;
	}

	.main__search {
		padding-left: 10px;
		height: 40px;
		margin-bottom: 0;
		border-radius: 15px;
	}

	.main__search-input {
		border-radius: 100px;
		height: 25px;
		max-width: 174px;
		width: 55%;
	}

	.section__search p {
		margin: 2px;
	}

	.search__icons--box {
		margin-top: 10px;
	}

	.icon__search {
		width: 70px;
		height: 70px;
	}
}