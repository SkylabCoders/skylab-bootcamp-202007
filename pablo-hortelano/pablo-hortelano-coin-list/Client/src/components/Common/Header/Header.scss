.MuiAppBar-colorPrimary {
  background-color: #ff3d6b !important;
}

.header__button {
  background-color: white !important;
  color: black !important;
}
