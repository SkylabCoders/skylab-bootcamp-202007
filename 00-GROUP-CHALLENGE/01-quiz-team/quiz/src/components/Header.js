import React from 'react';
import { NavLink } from 'react-router-dom';
import './../css/styles.css'
import './../css/Header.css'


function Header(props) {
	return (
		<>
			<div className="header border border-info">
				<div className="inner__header">
						<div className="logo__header">
							<img className="img__header" src="https://storage.needpix.com/rsynced_images/quiz-2074324_1280.png" alt="quiz logo" />
						</div>
					<nav>
						<NavLink activeClassName="my-active-class link" to="/">
							Home
				</NavLink>{' '}
						{' | '}
						<NavLink activeClassName="my-active-class link" to="/leaderboard">
							Leaderboard
				</NavLink>{' '}
						{' | '}
						<NavLink activeClassName="my-active-class link" to="/dashboard">
							Dashboard
				</NavLink>
					</nav>
					<NavLink activeClassName="my-active-class" to="/Login">
						<button type="button" class="btn btn-info">Login</button>
					</NavLink>
				</div>
			</div>
		</>
	)
}

export default Header;

