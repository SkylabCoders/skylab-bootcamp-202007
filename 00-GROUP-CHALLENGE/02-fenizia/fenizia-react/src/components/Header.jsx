import React, { useState, useEffect } from "react";
import { Link } from "react-router-dom";
import "./Header.css";
import cartStore from "../stores/cartStore";
// import TextInput from "./TextInput";

function Header() {
  // const [productName, setProductName] = useState("");
  const [showForm, setShowForm] = useState(false);
  const [genre, setGenre] = useState("");
  const [numberCart, setNumberCart] = useState(cartStore.getNumberCart());

  setInterval(() => setNumberCart(cartStore.getNumberCart()), 100);

  useEffect(() => {
    cartStore.addChangeListener(onChange);
    return () => cartStore.removeChangeListener(onChange);
  },[])

  function handleClick() {
    setShowForm(true);
  }

  function onChange(){
    setNumberCart(cartStore.getNumberCart()); 
  }
  
  return (
    <>
      {/* {
    searchForm = showForm ? (
      <TextInput
      name="products"
      value={productName}
      placeholder="¿Qué quieres buscar?"
      onChange={(event) => onFieldChange(event.target.value, setProductName)}
    />

    ) : 'Meck!';
    } */}

      <section className="main-nav">
        <div className="hamburger">
          <Link to="/" className="icon__hamburger"></Link>
          <ul className="hamburger__products">
            <Link
              onclick={() => setGenre("cocina")}
              to="/cocina"
              className="novels"
            >
              Cocina
            </Link>
            <Link
              onclick={() => setGenre("ensayo")}
              to="/ensayo"
              className="childish"
            >
              Ensayo
            </Link>
            <Link
              onclick={() => setGenre("historia")}
              to="/historia"
              className="childish"
            >
              Historia
            </Link>
            <Link
              onclick={() => setGenre("infantil")}
              to="/infantil"
              className="childish"
            >
              Infantil
            </Link>
            <Link
              onclick={() => setGenre("romantico")}
              to="/romantico"
              className="childish"
            >
              Romántico
            </Link>
            <Link
              onclick={() => setGenre("thriller")}
              to="/thriller"
              className="childish"
            >
              Thriller
            </Link>
            <Link
              onclick={() => setGenre("todos")}
              to="/todos"
              className="childish"
            >
              TODOS
            </Link>
          </ul>
        </div>

        <Link to="/" className="icon__logo"></Link>

        <div className="main-nav-text">
          <Link to="/">INICIO</Link>
          <Link to="/sobre-nosotros">SOBRE NOSOTROS</Link>

          <div className="categories">
            <Link>GÉNEROS</Link>
            <ul className="menu__products">
              <Link
                onclick={() => setGenre("cocina")}
                to="/cocina"
                className="novels"
              >
                Cocina
              </Link>
              <Link
                onclick={() => setGenre("ensayo")}
                to="/ensayo"
                className="childish"
              >
                Ensayo
              </Link>
              <Link
                onclick={() => setGenre("historia")}
                to="/historia"
                className="childish"
              >
                Historia
              </Link>
              <Link
                onclick={() => setGenre("infantil")}
                to="/infantil"
                className="childish"
              >
                Infantil
              </Link>
              <Link
                onclick={() => setGenre("romantico")}
                to="/romantico"
                className="childish"
              >
                Romántico
              </Link>
              <Link
                onclick={() => setGenre("thriller")}
                to="/thriller"
                className="childish"
              >
                Thriller
              </Link>
              <Link
                onclick={() => setGenre("todos")}
                to="/todos"
                className="childish"
              >
                TODOS
              </Link>
            </ul>
          </div>

          <a href="https://github.com/simonbesteiro/fenizia">CONTACTO</a>
          
        </div>

        {/* {searchForm} */}

        <Link to={handleClick} className="icon__search"></Link>

        <Link to="/login" className="icon__login"></Link>
        
        <Link to="/cart" className="icon__bag">
          <div className="icon__number">{numberCart}</div>
        </Link>
      
      </section>
    </>
  );
}

export default Header;
