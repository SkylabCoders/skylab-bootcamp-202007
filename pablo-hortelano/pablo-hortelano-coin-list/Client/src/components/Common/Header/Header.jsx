import React from "react";
import LogOut from "../../LogOut/LogOut";
import { makeStyles } from "@material-ui/core/styles";
import AppBar from "@material-ui/core/AppBar";
import Toolbar from "@material-ui/core/Toolbar";
import Typography from "@material-ui/core/Typography";
import IconButton from "@material-ui/core/IconButton";
import { NavLink } from "react-router-dom";
import clsx from "clsx";
import Drawer from "@material-ui/core/Drawer";
import Button from "@material-ui/core/Button";
import List from "@material-ui/core/List";
import Divider from "@material-ui/core/Divider";
import ListItem from "@material-ui/core/ListItem";
import ListItemIcon from "@material-ui/core/ListItemIcon";
import ListItemText from "@material-ui/core/ListItemText";

import HomeIcon from "@material-ui/icons/Home";
import PersonIcon from "@material-ui/icons/Person";
import MeetingRoomIcon from "@material-ui/icons/MeetingRoom";
import "./Header.scss";
import { useAuth0 } from "@auth0/auth0-react";

const useStyles = makeStyles((theme) => ({
  root: {
    flexGrow: 1,
  },
  menuButton: {
    marginRight: theme.spacing(2),
  },
  title: {
    flexGrow: 1,
  },
  list: {
    width: 250,
  },
  fullList: {
    width: "auto",
  },
}));

export default function Header() {
  const { isAuthenticated } = useAuth0();
  const classes = useStyles();
  const [state, setState] = React.useState({
    right: false,
  });

  const toggleDrawer = (anchor, open) => (event) => {
    if (
      event.type === "keydown" &&
      (event.key === "Tab" || event.key === "Shift")
    ) {
      return;
    }

    setState({ ...state, [anchor]: open });
  };

  const list = (anchor) => (
    <div
      className={clsx(classes.list, {
        [classes.fullList]: anchor === "top" || anchor === "bottom",
      })}
      role="presentation"
      onClick={toggleDrawer(anchor, false)}
      onKeyDown={toggleDrawer(anchor, false)}
    >
      <List>
        <NavLink
          to={{
            pathname: "/",
          }}
        >
          <ListItem button>
            <ListItemIcon>
              <HomeIcon />
            </ListItemIcon>
            <ListItemText primary={"Landing"} />
          </ListItem>
        </NavLink>
      </List>
      <List>
        <NavLink
          to={{
            pathname: "/Profile",
          }}
        >
          <ListItem button>
            <ListItemIcon>
              <PersonIcon />
            </ListItemIcon>
            <ListItemText primary={"Profile"} />
          </ListItem>
        </NavLink>
      </List>
      <Divider />
      <List>
        <ListItem button>
          <ListItemIcon>
            <MeetingRoomIcon />
          </ListItemIcon>
          <LogOut />
        </ListItem>
      </List>
      <Divider />
    </div>
  );

  return (
    (isAuthenticated && (
      <>
        <div></div>
        <div className={classes.root}>
          <AppBar position="static" className="test">
            <Toolbar>
              <IconButton
                edge="start"
                className={classes.menuButton}
                color="inherit"
                aria-label="menu"
              ></IconButton>
              <Typography variant="h6" className={classes.title}>
                CoinList
              </Typography>
              {["right"].map((anchor) => (
                <React.Fragment key={anchor}>
                  <Button
                    onClick={toggleDrawer(anchor, true)}
                    className="header__button"
                  >
                    MENU
                  </Button>
                  <Drawer
                    anchor={anchor}
                    open={state[anchor]}
                    onClose={toggleDrawer(anchor, false)}
                  >
                    {list(anchor)}
                  </Drawer>
                </React.Fragment>
              ))}
            </Toolbar>
          </AppBar>
        </div>
      </>
    )) || (
      <>
        <div></div>
        <div className={classes.root}>
          <AppBar position="static" className="test">
            <Toolbar>
              <IconButton
                edge="start"
                className={classes.menuButton}
                color="inherit"
                aria-label="menu"
              ></IconButton>
              <Typography variant="h6" className={classes.title}>
                CoinList
              </Typography>
              {["right"].map((anchor) => (
                <React.Fragment key={anchor}>
                  <Drawer
                    anchor={anchor}
                    open={state[anchor]}
                    onClose={toggleDrawer(anchor, false)}
                  >
                    {list(anchor)}
                  </Drawer>
                </React.Fragment>
              ))}
            </Toolbar>
          </AppBar>
        </div>
      </>
    )
  );
}
