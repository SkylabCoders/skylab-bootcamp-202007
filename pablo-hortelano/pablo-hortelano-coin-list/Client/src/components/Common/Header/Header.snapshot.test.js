import renderer from "react-test-renderer";
import Header from "./Header";
import React from "react";

describe("HeaderNavbar snapshot", () => {
  const tree = renderer.create(<Header />);
  it("should match", () => {
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
