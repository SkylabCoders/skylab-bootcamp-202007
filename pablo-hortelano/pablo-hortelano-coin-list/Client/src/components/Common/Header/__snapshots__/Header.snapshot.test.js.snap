// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderNavbar snapshot should match 1`] = `
Array [
  <div />,
  <div
    className="makeStyles-root-1"
  >
    <header
      className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary test MuiPaper-elevation4"
    >
      <div
        className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <button
          aria-label="menu"
          className="MuiButtonBase-root MuiIconButton-root makeStyles-menuButton-2 MuiIconButton-colorInherit MuiIconButton-edgeStart"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiIconButton-label"
          />
        </button>
        <h6
          className="MuiTypography-root makeStyles-title-3 MuiTypography-h6"
        >
          CoinList
        </h6>
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-text header__button"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            MENU
          </span>
        </button>
      </div>
    </header>
  </div>,
]
`;
